internal/provider/scheduler/request.go

```diff
@@ -0,0 +1,47 @@
+// Package scheduler provides a simple scheduler interface to request deployments on nodes.
+package scheduler
+
+import (
+	proxyTypes "github.com/threefoldtech/grid_proxy_server/pkg/types"
+)
+
+var (
+	statusUP = "up"
+	trueVal  = true
+)
+
+// Request struct for requesting a capacity
+type Request struct {
+	Cap       Capacity
+	Name      string
+	Farm      string
+	HasIPv4   bool
+	HasDomain bool
+	Certified bool
+
+	farmID int
```

Comment:
this field seems suspicious, not sure who sets it or at which point, it seems to be enforced in tests as well.

Reply:
apparently 
![image](https://user-images.githubusercontent.com/64129/210376901-b715d7f5-02cb-48fb-bf1d-3e9dfa967eb6.png)

it's enforced in the tests because they don't know the Farm name ahead of time, and if the farm name exists they resolve into an ID

Reply:
![image](https://user-images.githubusercontent.com/64129/210377322-2c56ffe3-7e1f-44aa-ae83-2d058711f7ad.png)

according to @rawdaGastan that check seems the main usecase